feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go

```diff
@@ -171,13 +171,14 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 	isisLevel2 := isis.GetOrCreateLevel(2)
 	isisLevel2.MetricStyle = oc.Isis_MetricStyle_WIDE_METRIC
 
-	if !deviations.ISISLevelAuthenticationNotRequired(dut) {
-		isisLevel2Auth := isisLevel2.GetOrCreateAuthentication()
-		isisLevel2Auth.Enabled = ygot.Bool(true)
-		isisLevel2Auth.AuthPassword = ygot.String(authPassword)
-		isisLevel2Auth.AuthMode = oc.IsisTypes_AUTH_MODE_MD5
-		isisLevel2Auth.AuthType = oc.KeychainTypes_AUTH_TYPE_SIMPLE_KEY
-	}
+	isisLevel2Auth := isisLevel2.GetOrCreateAuthentication()
+	isisLevel2Auth.Enabled = ygot.Bool(true)
+	isisLevel2Auth.DisableCsnp = ygot.Bool(false)
```

Comment:
From https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html, disable-csnp, disable-lsp and disable-psnp are set to false(default).

Pls add a deviation if below flags need to be explicitly set to false

isisLevel2Auth.DisableCsnp = ygot.Bool(false)
isisLevel2Auth.DisableLsp = ygot.Bool(false)
isisLevel2Auth.DisablePsnp = ygot.Bool(false)

Reply:
Added `ISISExplicitLevelAuthenticationConfig` deviation to configure these flag explicitly.  Also updated PR heading and description accordingly.